src/mongo/db/storage/storage_engine_init.cpp

```diff
@@ -271,7 +273,15 @@ StorageEngine::LastShutdownState initializeStorageEngine(OperationContext* opCtx
                       storageGlobalParams,
                       encryptionGlobalParams.kmipKeyIds.encryption,
                       initFlags);
-        throw;
+        if (!encryptionGlobalParams.kmipServerName.empty()) {
+            LOGV2(29111,
+                  "Rotated master encryption key",
+                  "oldMasterKeyId"_attr = encryptionGlobalParams.kmipKeyIds.decryption,
+                  "newMasterKeyId"_attr = encryptionGlobalParams.kmipKeyIds.encryption);
+        } else {
+            LOGV2(29111, "Rotated master encryption key");
```

Comment:
Don't forget to update log ID [list](https://confluence.percona.com/display/PSMDB/Document+logging+subsystem+identifiers+used+in+Percona+code) after merging.